celements-xwiki-core/src/main/java/com/xpn/xwiki/XWiki.java

```diff
@@ -1598,7 +1597,8 @@ public String getBaseSkin(String skin, XWikiContext context) {
   }
 
   public String getSpaceCopyright(XWikiContext context) {
-    String defaultValue = "Copyright 2004-" + Calendar.getInstance().get(Calendar.YEAR) + " XWiki";
+    String defaultValue = "Copyright 2007-" + Calendar.getInstance().get(Calendar.YEAR)
+        + " Synventis";
     return getSpacePreference("webcopyright", defaultValue, context);
```

Comment:
do we actually still need this space preference "webcopyright"?

Reply:
yes,, is shown in Albatross Footer... Maybe we should replace with a reference to LGPL License on github?